Adapter.C

```diff
@@ -110,6 +110,10 @@ bool preciceAdapter::Adapter::configFileRead()
                     }
                     DEBUG(adapterInfo("    connectivity : " + std::to_string(interfaceConfig.meshConnectivity)));
 
+                    // Reset the displacement when defining the interface
+                    interfaceConfig.resetDisplacement = interfaceDict.lookupOrDefault<bool>("resetDisplacement", true);
```

Comment:
This also needs to be documented in https://github.com/precice/openfoam-adapter/blob/develop/docs/config.md#advanced-configuration

Reply:
added